scirpy/_plotting/_clonotype_imbalance.py

```diff
@@ -101,8 +98,20 @@ def clonotype_imbalance(
         return volcano(df, **kwargs)
 
     else:
-        if fig_kws is None:
-            fig_kws = {"alpha": 0.7, "dodge": True}
+        if "sns_kws" in kwargs:
```

Comment:
In that case I would get rid of the "sns_kws" as well. I don't think it's needed and atm it's not even documented, so there is no way a user would know he could pass a `sns_kws` arument also.

Reply:
Ok, I will just pass on the kwargs to seaborn